src/main/java/org/sagebionetworks/bridge/upload/UploadValidationTask.java

```diff
@@ -88,6 +88,8 @@ public void run() {
         // Create temp dir.
         File tempDir = fileHelper.createTempDir();
         context.setTempDir(tempDir);
+        String tempDirPath = tempDir.getAbsolutePath();
+        logger.info("Created temp dir " + tempDirPath);
```

Comment:
shouldn't this right under the line where the dir is created,  `File tempDir = fileHelper.createTempDir()`?

Reply:
I'm not sure that matters if the only other place the variable is referenced is the try/catch block below. Also, this is a PR merged by Dwayne that was never promoted from dev to uat and I'd rather not change it.